stable/joomla/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: joomla
-version: 1.0.3
+version: 1.0.4
```

Comment:
@tompizmor implemented similar changes for `moodle` and `prestashop`. @sameersbn includes some comments about that https://github.com/kubernetes/charts/pull/5733/files#r190521131

I think we should have a consensus on this and apply it in the same way to the different charts.

Reply:
Increased the minor version